src/components/Footer/Footer.js

```diff
@@ -113,6 +113,33 @@ export const Footer = () => {
             </a>
           </div>
         </div>
+        <div className="linksWrapper">
```

Comment:
Tutaj Sylwia proponowałbym zaimplementować to w taki sposób, aby te linki były generowane ze zmiennej, która mogłaby być zaimplementowana na górze komponentu. Zmienna mogła by mieć taką formę:

```
const LINKS = [
{
  title: t('home')
  to: "/",
  arialLabel: t('ariaLabelHomePage')
}, 
{
  title: t('news')
  to: "/news",
  arialLabel: t('ariaLabelNews')
},
itd...
]
```

A potem w komponencie każdy link byłby generowany za pomocą `map` czyli:
```
LINKS.map(({title, to, arialLabel}) => (
<li>
  <Link to={to} aria-label={arialLabel}>
    {title}
  </Link>
</li>
))
```

W taki sposób mamy możliwość szybkiej i prostej zmiany linków i jest fajnie przejrzyste